strings/tips.txt

```diff
@@ -18,13 +18,16 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
 Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
-On some maps, you can use a machine in the vault to deposit space cash for cargo points or rob cargo blind.
+On all maps, you can use a machine in the vault to deposit space cash for cargo points or rob cargo, but if you print out money it blares a alarm to the crew!
```

Comment:
```suggestion
On all maps, you can use a machine in the vault to deposit space cash for cargo points. Otherwise, use it to steal the station's cash and get out before the alarm goes off.
```